packages/auto-label/repos.txt

```diff
@@ -0,0 +1 @@
+testOwner/testRepo
```

Comment:
Kind of a weird thing to have hanging out in the root of the project :) I'm curious what this will be used for!